apps/frontend/src/components/DashBoard.tsx

```diff
@@ -353,20 +353,20 @@ const Dashboard = () => {
             path="/changeRole"
             component={ChangeRole}
           />
-          {isSEPEnabled && (
+          {isFapEnabled && (
             <TitledRoute
               setHeader={setHeader}
-              title={t('SEP')}
-              path="/SEPPage/:id"
-              component={SEPPage}
+              title={t('Fap')}
```

Comment:
There is the function I was referring to sorry, they take the corresponding entry in the translation.json file and insert the corriponding string. So this `t()` can be removed and just to leave `Fap`. I think since it cannot find a entry in translation.json it just returns `Fap` in this case.

Reply:
I get it now. Thanks for the clarification.

Reply:
@TCMeldrum it should be updated now and you can doublecheck.